src/expressions/operator-expr.md

```diff
@@ -385,8 +384,19 @@ same trait object.
 * `u8` to `char` cast
     * Casts to the `char` with the corresponding code point.
 
+\* if integer-to-float casts with this rounding ode and overflow behavior are
+not supported natively by the hardware, these casts will likely be slower than
+expected.
+
+\*\* if f64-to-f32 casts with this rounding mode and overflow behavior are not
+supported natively by the hardware, these casts will likely be slower than
+expected.
+
+\*\*\* as defined in IEEE 754-2008 &sect;4.3.1: pick the nearest floating point
```

Comment:
Nit: might be a bit more readable with numbered footnotes?

Reply:
I agree, but this style of footnotes is used elsewhere in this chapter and in some other chapters too, and both deviating locally and updating them all seems ungreat.

Reply:
Oh I see; guess I didn't search thoroughly enough to tell what the overall style is. :)